pages/madewithcode.jsx

```diff
@@ -0,0 +1,82 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+var OutboundLink = require('react-ga').OutboundLink;
+
+var config = require('../lib/config');
+
+var Intro = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <h1>Made with Code</h1>
+        <section className="intro">
+          <Illustration
+          width={200} height={355}
+          src1x="/img/pages/madewithcode/logo_mwc.png"
```

Comment:
I think we are still doing `src1x` (see [here](https://github.com/mmmavis/teach.mozilla.org/blob/3745a26e3dcb4216d1886d9ea852c6f9bb7d5971/components/illustration.jsx#L19)) and the underlying <ImageTag> will [map the path(s) to `src` and `srcSet`](https://github.com/mmmavis/teach.mozilla.org/blob/3745a26e3dcb4216d1886d9ea852c6f9bb7d5971/components/imagetag.jsx#L22-L23)